CHANGELOG

```diff
@@ -1,3 +1,13 @@
+Snowflake Web Version 1.0.0 (2022-02-24)
```

Comment:
I think the version number here is wrong

Reply:
thanks, you this has been revised now.